build-tools-internal/build.gradle

```diff
@@ -238,6 +212,9 @@ dependencies {
   api localGroovy()
   api gradleApi()
 
+  api 'org.elasticsearch:build-conventions:1.0'
```

Comment:
Where's the 1.0 version coming from? Shouldn't this be the same as the Elasticsearch version?